app/classes/Customer.py

```diff
@@ -0,0 +1,69 @@
+import json
+from dataclasses import dataclass
+from math import sqrt
+from datetime import datetime
+
+with open("../app/config.json", "r") as f:
```

Comment:
Wrong path to the config, the tests are failes

Reply:
this is the only way, how I can run tests in venv, I tried "app.config.json" and "config.json" and both cases doesn't work